database/models/user.model.js

```diff
@@ -0,0 +1,65 @@
+import { model, Schema, Types } from "mongoose";
+
+const schema = new Schema(
+  {
+
+    name: {
+      type: String,
+      maxLength: [100, "name can't be more than 100 characters"],
+      required: [true, "name is required"],
+      trim: true,
+    },
+    email: {
+      type: String,
+      unique: true,
+      maxLength: [100, "email can't be more than 100 characters"],
+      minLength: [3, "email can't be less than 3 characters"],
+      required: [true, "email is required"],
+      trim: true,
+    },
+    password: {
+      type: String,
+      maxLength: [100, "password can't be more than 100 characters"],
```

Comment:
delete max
add  minLength: [6, "Password must be at least 6 characters long"],

add select:false